src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -29,8 +30,9 @@ public class DeleteCommandTest {
 
     @Test
     public void execute_validIndexUnfilteredList_success() {
+        String dummyName = " ";
```

Comment:
Since dummyName is used multiple times throughout this file, it could be declared as a static variable for reuse?